code/modules/clothing/masks/gasmask.dm

```diff
@@ -14,6 +14,50 @@
 		"Vox" = 'icons/mob/species/vox/mask.dmi'
 		)
 
+// **** Welding gas mask ****
+
+/obj/item/clothing/mask/gas/welding
+	name = "welding mask"
+	desc = "A gas mask with built in welding goggles and face shield. Looks like a skull, clearly designed by a nerd."
+	icon_state = "weldingmask"
+	item_state = "weldingmask"
+	m_amt = 3000
+	g_amt = 1000
+	var/up = 0
+	flash_protect = 2
+	tint = 2
+	armor = list(melee = 10, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 0)
+	origin_tech = "materials=2;engineering=2"
+	action_button_name = "Toggle Welding Helmet"
+
+/obj/item/clothing/mask/gas/welding/attack_self()
+	toggle()
+
+
+/obj/item/clothing/mask/gas/welding/verb/toggle()
+	set category = "Object"
+	set name = "Adjust welding mask"

```

Comment:
I'm 90% sure that nothing else has it's adjust verb anymore, and it's supposed to only have an action button.